docs/src/get-started.md

```diff
@@ -0,0 +1,191 @@
+# Getting Started
+
+## Installation
+
+To use Turing, you need install Julia first and then install Turing.
+
+### Install Julia
+
+You will need Julia 0.6, which you can get from [the official Julia website](http://julialang.org/downloads/).
+
+It provides three options for users
+
+1. A command line version [Julia/downloads](http://julialang.org/downloads/) (**recommended**)
+2. A community maintained IDE [Juno](http://www.junolab.org/)
+3. [JuliaBox.com](https://www.juliabox.com/) - a Jupyter notebook in the browser
```

Comment:
Em-dash here as space-hypen-space. Maybe space–en-dash–space to be consistent with the above comment.